pybaseball/statcast_batter.py

```diff
@@ -33,5 +33,26 @@ def statcast_batter_exitvelo_barrels(year: int, minBBE: str = "q") -> pd.DataFra
     url = f"https://baseballsavant.mlb.com/leaderboard/statcast?type=batter&year={year}&position=&team=&min={minBBE}&csv=true"
     res = requests.get(url, timeout=None).content
     data = pd.read_csv(io.StringIO(res.decode('utf-8')))
+    return data
 
+@cache.df_cache()
+def statcast_batter_expected_stats(year: int, minPA: str = "q") -> pd.DataFrame:
```

Comment:
nit-pick: isn't this `Union[int, str]`?

Reply:
Do you mean for `year`?

Reply:
(minPA, to be clear)

Reply:
Cool. I'll also add that for `statcast_batter_exitvelo_barrels()` since I think it would be applicable there, too